Sources/Turf/GeoJSON.swift

```diff
@@ -98,11 +98,26 @@ public protocol ForeignMemberContainer {
     var foreignMembers: JSONObject { get set }
 }
 
+/**
+ Key to pass to populate a `userInfo` dictionary, which is passed to the `JSONDecoder` or `JSONEncoder` to enable processing foreign members.
+*/
+public extension CodingUserInfoKey {
+    /**
+     Indicates if coding of foreign members is enabled.
+     
+     Boolean flag to enable coding. Default (or missing) value is to ignore foreign members.
+     */
+    static let allowForeignMembersCoding = CodingUserInfoKey(rawValue: "com.mapbox.turf.coding.allowForeignMembersCodingKey")!
```

Comment:
`includesForeignMembers` would be more succinct.

Reply:
Updated